internal/core/software_renderer/minimal_software_window.rs

```diff
@@ -45,6 +45,27 @@ impl MinimalSoftwareWindow {
         }
     }
 
+    #[rustversion::since(1.85)]
+    #[cfg(feature = "experimental")]
```

Comment:
```suggestion
    #[cfg(feature = "experimental")]
    #[rustversion::since(1.85)]
```

otherwise this might not compile if experimental feature is not enabled, i think

Reply:
Good idea :)